### PR TITLE
f-checkout@0.174.0 - Checkout AUS/NZ copy in tenant files

### PR DESCRIPTION
**Pull request by charliearlie**
### Added
- Add copy for AU/NZ locales

### CODE REVIEW

**Comment by matthewhardern** [comment]
@charliearlie The bundle watch error above was due to an unfortunate side effect of this build running as I was switching over to the new build Circle CI process. I have a PR that will fix this issue open atm, I will then re trigger your build and it should go through fine.

**Comment by charliearlie** [comment]
Thanks @matthewhardern

**Comment by matthewhardern** [comment]
> Thanks @matthewhardern

It looks like its using master cache now anyway so all working as it should

**Comment by matthewhardern** [comment]
@charliearlie all fixed now

File: packages/components/organisms/f-checkout/CHANGELOG.md [hunk 1]
```diff
@@ -20,6 +20,15 @@ v0.173.1
 - Exports in test selector file.
 
 
+v0.174.0
+------------------------------
+*August 18, 2021*
```
Comment (DamianMullins) [inline packages/components/organisms/f-checkout/CHANGELOG.md]: Need to bump the date.

Comment (charliearlie) [reply packages/components/organisms/f-checkout/CHANGELOG.md]: Done 🙂

**Review by DamianMullins — DISMISSED** [review]

**Review by kevinrodrigues — DISMISSED** [review]

**Review by kevinrodrigues — DISMISSED** [review]

File: packages/components/organisms/f-checkout/src/tenants/en-AU.js [hunk 2]
```diff
@@ -1,4 +1,223 @@
...
+        },
+        addressLine1: {
+            requiredError: 'Please enter the first line of your address'
+        },
+        locality: {
+            requiredError: 'Please enter your town or city'
+        },
+        postcode: {
+            requiredError: 'Please enter your full UK postcode',
+            invalidCharError: 'This doesn’t look like a UK postcode, can you enter it again please?'
```
Comment (JoshuaNg2332) [inline packages/components/organisms/f-checkout/src/tenants/en-AU.js]: Shouldn't this to be changed to AU postcode?

Comment (charliearlie) [reply packages/components/organisms/f-checkout/src/tenants/en-AU.js]: Good spot! 😬

File: packages/components/organisms/f-checkout/src/tenants/en-AU.js [hunk 3]
```diff
@@ -1,4 +1,223 @@
...
+            body: 'Please check the day and time above'
+        }
+    },
+
+    buttonText: 'Go to payment',
+
+    userNote: {
+        delivery: {
+            title: 'Leave a note',
+            text: 'Leave a note for the restaurant with anything they need to know (e.g. the doorbell doesn\'t work). Do not include details about any allergies here.',
```
Comment (minrwhite) [inline packages/components/organisms/f-checkout/src/tenants/en-AU.js:148]: Proper quote marks here? Or not desirable as an example of proper user input?
```suggestion
            text: 'Leave a note for the restaurant with anything they need to know (e.g. the doorbell doesn’t work). Do not include details about any allergies here.',
```

File: packages/components/organisms/f-checkout/src/tenants/en-NZ.js [hunk 4]
```diff
@@ -1,4 +1,223 @@
...
+    checkoutTermsAndConditions: 'By placing an order you agree to our {termsAndConditions}. Please read our {privacyPolicy} and {cookiePolicy}.',
+
+    termsAndConditionsLinkText: 'Terms and Conditions',
+    termsAndConditionsLinkUrl: 'https://www.menulog.com.au/info/privacy-policy',
+
+    privacyPolicyLinkText: 'Privacy Policy',
+    privacyPolicyLinkUrl: 'https://www.menulog.com.au/info/privacy-policy',
+
+    cookiePolicyLinkText: 'Cookie Policy',
+    cookiePolicyLinkUrl: 'https://www.menulog.com.au/info/privacy-policy#cookies_policy',
```
Comment (JoshuaNg2332) [inline packages/components/organisms/f-checkout/src/tenants/en-NZ.js]: Do these links need to go to menulog.com.nz as this is for the NZ tenant?

File: packages/components/organisms/f-checkout/src/tenants/en-NZ.js [hunk 5]
```diff
@@ -1,4 +1,223 @@
...
+            body: 'Please check the day and time above'
+        }
+    },
+
+    buttonText: 'Go to payment',
+
+    userNote: {
+        delivery: {
+            title: 'Leave a note',
+            text: 'Leave a note for the restaurant with anything they need to know (e.g. the doorbell doesn\'t work). Do not include details about any allergies here.',
```
Comment (minrwhite) [inline packages/components/organisms/f-checkout/src/tenants/en-NZ.js:148]: As above, proper quote marks here? Or not desirable as an example of proper user input?
```suggestion
            text: 'Leave a note for the restaurant with anything they need to know (e.g. the doorbell doesn’t work). Do not include details about any allergies here.',
```

**Review by minrwhite — DISMISSED** [review]
A couple of minor points, but happy to approve if these can be addressed later

File: packages/components/organisms/f-checkout/src/tenants/en-NZ.js (hunk 4 above)
Comment (charliearlie) [reply packages/components/organisms/f-checkout/src/tenants/en-NZ.js]: I need a lay down. No idea how I didn't spot that too. Updated now 🙂

File: packages/components/organisms/f-checkout/src/tenants/en-AU.js (hunk 3 above)
Comment (charliearlie) [reply packages/components/organisms/f-checkout/src/tenants/en-AU.js:148]: These are lifted straight from the UK file so definitely something that can be looked at to change in all files.

**Review by JoshuaNg2332 — APPROVED** [review]

**Review by minrwhite — APPROVED** [review]

**Pull request merged**